### DISCUSSION TITLE
In-game voting système

### DISCUSSION THREAD

**Discussion by NomiceBinocle** [sunflower-land/sunflower-land#2620] [Polls]
Dear developer and fellow Sunflower community,

As a passionate player of Sunflower, I would like to present a proposal that I hope will pique your interest. I have been thinking about the possibility of introducing an in-game voting system to our beloved game, and I believe it could truly enhance our community engagement and improve our collaboration with the development team.

Imagine a system where every player has the opportunity to vote directly from within the game to influence development decisions. We could express our opinions and actively participate in shaping the future of Sunflower. Whether it's for new features, significant changes, or special events, our voices would truly matter.

I firmly believe that this approach would enable developers to be more aligned with the community's expectations and desires. By granting us the ability to actively participate in decisions, you would also foster a stronger sense of ownership and greater player satisfaction.

I invite all members of the Sunflower community to voice their opinions on this proposal. Share your thoughts in the comments below and let us know if you would be in favor of introducing an in-game voting system. Your support and participation will be crucial in advancing this idea.

Together, we can create an even more exceptional Sunflower that truly reflects the community's wishes. Thank you all for your attention and involvement.

**Comment by brian-man** [comment]
Strawman
1. document the impending "Vote 1" at least a week ahead
2. introduce a voting UI in game in the form of "buy A, B, or C" using "Vote 1 Ticket" (could be a new collection, doesn't need to be in SLI or saved to blockchain)
3. airdrop via mailbox a single "Vote 1 Ticket" to all farms

You automatically get only active and somewhat awake players.
No discord requirement.

**Reply by NomiceBinocle** [reply]
simple and effective all we need, maybe establish a ticket scale distributed according to the number of expansions to encourage players to evolve to have more impact
